.isort.cfg

```diff
@@ -38,6 +38,7 @@ extend_skip=
     chia/data_layer/data_store.py
     chia/data_layer/dl_wallet_store.py
     chia/data_layer/download_data.py
+    chia/data_layer/util/benchmark.py
```

Comment:
I apologize for not realizing this was a rename when I complained about the isort'ing.  So two choices from here now that I get that.  Regardless we should delete the old name from this file since it is no longer needed.  From there you can remove this line too and do the isort now that I get that it is localized and there's a reason for it, or just leave this be.  Maybe next time the whole move can be a separate PR since it seem unrelated.  This is fine for now though.